tox.ini

```diff
@@ -0,0 +1,27 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py35, docs
+
+[testenv]
+commands =
+    coverage run -m nose
```

Comment:
We should make sure to tag the uploads to codecov with the version of python that they were generated on.